src/Microsoft.ML.DnnAnalyzer/Microsoft.ML.DnnAnalyzer/Microsoft.ML.DnnAnalyzer.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <AssemblyName>DnnAnalyzer</AssemblyName>
```

Comment:
should it be part of TensorFlow package?

Reply:
It should, thanks for reminding.

---
In reply to: [218189010](https://github.com/dotnet/machinelearning/pull/862#discussion_r218189010) [](ancestors = 218189010)